system/ee/ExpressionEngine/View/design/system/index.php

```diff
@@ -2,18 +2,19 @@
 <div class="panel">
 	<?=form_open($form_url)?>
 
-  <div class="panel-heading">
+	<div class="panel-heading">
 		<div class="title-bar">
 			<h3 class="title-bar__title"><?=$cp_heading?></h3>
 		</div>
-  </div>
-
-    <div class="app-notice-wrap"><?=ee('CP/Alert')->getAllInlines()?></div>
+	</div>
+	<div class="design-system-template-body">
```

Comment:
Why do we need this new element? Shouldn't we place the table and alert into `panel-body` instead? Also, the indents for alert don't look right currently in Chrome
![image](https://user-images.githubusercontent.com/752126/166638108-be9f7db6-560a-4c37-86cf-aef6f3949fed.png)

Reply:
`.design-system-template-body` was added as an additional class for the main `system message templates`
`.single-templates-notice-wrapper` was added as an additional class for the single `system message templates` to fix spaces around the notice message.
if place the table and alert into `panel-body` it will break the current design
<img width="1440" alt="Screenshot 2022-05-04 at 10 38 11" src="https://user-images.githubusercontent.com/23382425/166652692-a2374380-3fe6-497a-bfaa-298be3ba8fcf.png">

Reply:
Then maybe question for @ajohnson 

Shouldn't we use same UI here which is there for the template, template variables and partials? I.e. with the left sidebar.

Also it looks like Email templates and Member templates (if enabled) need similar updates

Reply:
@intoeetive @Yulyaswan Correct, the `panel-body` should not be used when displaying a table list like the System Message Templates.

So the only inconsistency I'm noticing is the placement of the `app-notice` confirmation. When saving a template we place the confirmation within the `panel-heading` above the title like this:

<img width="616" alt="Screen Shot 2022-05-18 at 9 50 33 AM" src="https://user-images.githubusercontent.com/194579/169059069-e3f1323e-4546-4ff9-bf08-09de897d7bd7.png">

But with the System Message Templates we're displaying the `app-notice` confirmation below the `panel-heading` like this:

<img width="702" alt="Screen Shot 2022-05-18 at 9 50 48 AM" src="https://user-images.githubusercontent.com/194579/169059498-f71df966-1785-4248-93d5-9c9ca9360880.png">

So it'd be nice to change the location of the System Message Templates `app-notice` to be displayed in the `panel-heading` to match Templates.

But let me know if I'm misunderstanding your question @intoeetive . Thanks!

Reply:
@ajohnson I moved notification block above the title
https://packettide.atlassian.net/browse/EECORE-1792?focusedCommentId=16204